src/boot/efi/stub.c

```diff
@@ -26,6 +26,22 @@ DECLARE_NOALLOC_SECTION(".sdmagic", "#### LoaderInfo: systemd-stub " GIT_VERSION
 
 DECLARE_SBAT(SBAT_STUB_SECTION_TEXT);
 
+static void combine_measured_flag(int *value, bool measured) {
+        assert(value);
+
+        /* Combine the "measured" flag in a sensible way: if we haven't measured anything yet, the first
+         * write is taken as is. Later writes can only turn off the flag, never on again. Or in other words,
+         * we eventually want to return true iff we really measured *everything* there was to measure.
```

Comment:
/iff/if/

Reply:
iff → if and only if.

https://en.wikipedia.org/wiki/If_and_only_if